lldb/source/Plugins/TypeSystem/Swift/TypeSystemSwiftTypeRef.cpp

```diff
@@ -3148,13 +3148,41 @@ TypeSystemSwiftTypeRef::GetTypeBitAlign(opaque_compiler_type_t type,
             AsMangledName(type));
   return {};
 }
+
 bool TypeSystemSwiftTypeRef::IsTypedefType(opaque_compiler_type_t type) {
-  return m_swift_ast_context->IsTypedefType(ReconstructType(type));
+  auto impl = [&]() {
+    using namespace swift::Demangle;
+    Demangler dem;
+    NodePointer node = GetDemangledType(dem, AsMangledName(type));
+    return node && node->getKind() == Node::Kind::TypeAlias;
+  };
+
+  VALIDATE_AND_RETURN(impl, IsTypedefType, type, (ReconstructType(type)),
+                      (ReconstructType(type)));
 }
+
 CompilerType
 TypeSystemSwiftTypeRef::GetTypedefedType(opaque_compiler_type_t type) {
-  return m_swift_ast_context->GetTypedefedType(ReconstructType(type));
+  auto impl = [&]() -> CompilerType {
+    using namespace swift::Demangle;
+    Demangler dem;
+    NodePointer node = GetDemangledType(dem, AsMangledName(type));
+    if (!node || node->getKind() != Node::Kind::TypeAlias)
+      return {};
+    auto pair = ResolveTypeAlias(m_swift_ast_context, dem, node);
+    if (pair.first) {
+      NodePointer type_node = dem.createNode(Node::Kind::Type);
+      type_node->addChild(pair.first, dem);
+      auto a = RemangleAsType(dem, type_node);
+      return a;
```

Comment:
did you mean to have leave this as a separate variable and assignment?

Reply:
Oh whoops! I was dumping its value, but forgot to remove the variable.